ls1intum/Artemis
# `Development`: Update about us page

### Checklist
#### General


#### Client

### Motivation and Context



The features mentioned on Artemis are outdated and do not include all available features. We have integrated the functions of Artemis (from Github) into the "About us" page.

In addition, we tried to keep the bullet points as short as possible.

### Description


- Added features
- Translated into German
- Made it reusable and expandable

### Steps for Testing

Prerequisites:
- 1 Instructor
- 2 Students
- 1 Programming Exercise with Complaints enabled

1. Log in to Artemis
2. Navigate to the About us page (`/about`)
3. Check the features -> [Github](https://github.com/ls1intum/Artemis)
4. Check if something is missing (or mistranslated)

### Review Progress





#### Code Review
#### Manual Tests

### Screenshots



<details>
<summary>German (1200px)</summary>

![image](https://user-images.githubusercontent.com/60005702/175573865-b65c3002-1042-4ed0-85b1-7cf32bda3dfb.png)
</details>

<details>
<summary>German (1430px)</summary>

![image](https://user-images.githubusercontent.com/60005702/175573963-af60f8fe-8d2b-4484-a385-e560eede54ee.png)
</details>

<details>
<summary>English (1200px)</summary>

![image](https://user-images.githubusercontent.com/60005702/175573910-c4acf102-d46d-47f5-963b-c1a640f8efae.png)
</details>
